xml/dm_registers.xml

```diff
@@ -101,26 +117,40 @@
             0 otherwise.
         </field>
         <field name="confstrptrvalid" bits="4" reset="Preset" access="R">
-            0: \RdmConfstrptrZero--\RdmConfstrptrThree hold information which
+            <value v="0" name="invalid">
+            \RdmConfstrptrZero--\RdmConfstrptrThree hold information which
             is not relevant to the configuration structure.
+            </value>
 
-            1: \RdmConfstrptrZero--\RdmConfstrptrThree hold the address of the
+            <value v="1" name="valid">
+            \RdmConfstrptrZero--\RdmConfstrptrThree hold the address of the
             configuration structure.
+            </value>
         </field>
         <field name="version" bits="3:0" reset="3" access="R">
-            0: There is no Debug Module present.
+            <value v="0" name="not present">
```

Comment:
"none" is a little more concise (but I'm OK with what you have, too).

Reply:
Might want to use names that are similar to the version numbers in the DTM section.

Reply:
Changed to none. I'm not sure what version numbers in the DTM section you're referring to.

Reply:
> I'm not sure what version numbers in the DTM section you're referring to.

dtmcs.version: https://github.com/riscv/riscv-debug-spec/pull/727/files#diff-7e8d764a69e8e81a55604e4793a25d1077853b7e5f267607e1ec58366ed7f6afL65

But there's no equivalent to "not present" so as long as value 15 has a consistent name on both then I'm happy.

Reply:
I think they do match now that I've incorporated your suggestion to name 15 "custom."